src/screens/StarterPack/Wizard/StepFeeds.tsx

```diff
@@ -59,7 +59,7 @@ export function StepFeeds({moderationOpts}: {moderationOpts: ModerationOpts}) {
       : undefined
 
   const {data: searchedFeeds, isFetching: isFetchingSearchedFeeds} =
-    useSearchPopularFeedsQuery({q: throttledQuery})
+    usePopularFeedsSearch({query: throttledQuery})
```

Comment:
With the addition of `placeholderData: keepPreviousData,` there was no difference here, so should be a clean swap.